packages/ui/src/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="icon" type="image/svg+xml" sizes="any" href="/favicon.svg" />
-    <title>Joystream Governance App</title>
+    <title></title>
```

Comment:
This will only remove the current title. But the issue requirement is to make the title change according to the page the viewer is in.

Reply:
True it's in response to https://github.com/Joystream/pioneer/issues/4403#issuecomment-1584362104 although probably not the best solution. Possibly misunderstood also:
We can keep the static title in index.html and replace it later on with `document.title` or [Helmet](https://github.com/nfl/react-helmet). wdyt @dmtrjsg